src/components/Navbar/index.js

```diff
@@ -1,8 +1,23 @@
-import React from 'react';
-import './styles.css';
+import React, { useState } from 'react';
+import LoggedOutLinks from './LinksGroup/LoggedOutLinks';
+import LoggedInLinks from './LinksGroup/LoggedInLinks';
+import './styles.scss';
 
 const Navbar = () => {
+    const [loggedIn, setLoggedIn] = useState(false);
     
+    return (
+        <nav className="navbar">
+            <img className="logo" src="https://placehold.co/205x34" alt="Logo" />
```

Comment:
For images that are decorative (logos, banners and such) is better to put an empty alt and a ARIA role of presentation.
Like so

`<img className="logo" src="https://placehold.co/205x34" alt='' role='presentation' />`